src/components/CreateVmWizard/steps/Storage.js

```diff
@@ -503,21 +517,32 @@ class Storage extends React.Component {
 
   handleCellChange (rowData, field, value) {
     const editingRow = this.state.editing[rowData.id]
-    if (field === 'size') {
-      if (!isNumber(value)) return
-
-      const { minDiskSizeInGiB, maxDiskSizeInGiB } = this.props
-      if (value >= minDiskSizeInGiB && value <= maxDiskSizeInGiB) {
-        delete editingRow.invalidSizeValue
-      } else {
-        editingRow.invalidSizeValue = true
-      }
-      value = +value * (1024 ** 3) // GiB to B
+    const editingErrors = {}
+    switch (field) {
+      case 'size':
+        if (!isNumber(value)) return
+
+        const { minDiskSizeInGiB, maxDiskSizeInGiB } = this.props
+        if (value >= minDiskSizeInGiB && value <= maxDiskSizeInGiB) {
+          delete editingRow.invalidSizeValue
+        } else {
+          editingRow.invalidSizeValue = true
+        }
+        value = +value * (1024 ** 3) // GiB to B
+        break
+      case 'name':
+        editingErrors.diskInvalidName = !isDiskNameValid(value)
+        console.log('editingErrors', editingErrors)
```

Comment:
Don't you need to delete this console.log line? I can't see the benefit of leaving it.